calico_kubernetes/calico_kubernetes.py

```diff
@@ -146,35 +143,29 @@ def _configure_profile(self, endpoint):
 
         Currently assumes one pod with each name.
         """
-        pod = self._get_pod_config()
-
         logger.info('Configuring Pod Profile: %s', self.profile_name)
 
         if self._datastore_client.profile_exists(self.profile_name):
-            logger.error("Profile with name %s already exists, exiting.",
-                         self.profile_name)
-            sys.exit(1)
+            logger.warning("Profile with name %s already exists.",
```

Comment:
Aren't we going to hit this over and over again for each pod, since they're all going to be using the same profile?